src/server/transaction.h

```diff
@@ -431,13 +431,13 @@ class Transaction {
     bool concluding = false;
 
     unsigned cmd_seq_num = 0;  // used for debugging purposes.
+    unsigned shard_journal_cnt;
 
     // The shard_journal_write vector variable is used to determine the number of shards
     // involved in a multi-command transaction. This information is utilized by replicas when
     // executing multi-command. For every write to a shard journal, the corresponding index in the
     // vector is marked as true.
     absl::InlinedVector<bool, 4> shard_journal_write;
-    unsigned shard_journal_cnt;
   };
```

Comment:
Also shard_journal_cnt is the sum of shard_journal_write, that's why they're together. Sadly there's no comment and the existing one is too long 😅